src/plays/cryptocurrency-tracker/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Cryptocurrency Tracker
+
```

Comment:
Add some details of this play. Think about it as something a beginner will read and learn from it.